### DISCUSSION TITLE
Telebot to request and query data inside S3 (minIO) bucket

### DISCUSSION THREAD

**Discussion by d3v4mpz** [minio/minio#16702] [Q&A] [answered]
Greetings everyone, I'm relatively inexperienced in the realm of MinIO and I'm currently working on creating a telebot that can request and search for structured data (csv, xlsx) stored in a MinIO Bucket. Although I have a code sample, it's producing errors. I would greatly appreciate any other guidance or step-by-step instructions on how to proceed.

[code_sample_telebot_minio.txt](https://github.com/minio/minio/files/10820075/code_sample_telebot_minio.txt)

below is error

_File "C:\Users\PycharmProjects\pythonProject\venv\lib\site-packages\minio\helpers.py", line 399, in __init__
    raise ValueError("path in endpoint is not allowed")
**ValueError: path in endpoint is not allowed**
Exception ignored in: <function Minio.__del__ at 0x0000018BA6F73010>
Traceback (most recent call last):
  File "C:\Users\PycharmProjects\pythonProject\venv\lib\site-packages\minio\api.py", line 155, in __del__
    self._http.clear()
AttributeError: 'Minio' object has no attribute '_http'
Process finished with exit code 1_

**Comment by klauspost** [comment, marked as answer]
I don't know Python, but I'd guess `http://a.b.c.41:9000` -> `a.b.c.41:9000` is what it want.

Also, you specify http, but set `secure=True`. This will use TLS/HTTPS.
